src/main/java/spoon/reflect/declaration/CtType.java

```diff
@@ -59,11 +59,10 @@
 
 	/**
 	 *
-	 * NEVER USE THIS.
+	 * This is a low-level feature, it should never been used by non advanced users.
```

Comment:
Maybe highlight the risk part a bit more. Personally, I started ignoring warnings like `low-level feature`, as projects started writing this all over the code.

Reply:
feel free to Github-suggest or edit directly :)

Reply:
How about copying the paragraph from `CtTypeReference`? I know duplicates are bad, but we really don't change our Javadoc that often.

Reply:
I'm in favor of this, this is the best way to get incorrect documentation with drifting.